app/routes/events/components/EventList.js

```diff
@@ -123,11 +123,19 @@ const EventList = (props: EventListProps) => {
     <div className={styles.root}>
       <Helmet title="Arrangementer" />
       <Toolbar actionGrant={props.actionGrant} />
-      <EventListGroup name="Denne uken" events={events.currentWeek} />
+      {events.currentWeek && (
+        <EventListGroup name="Denne uken" events={events.currentWeek} />
```

Comment:
If you check inside the `<EventListGroup` component, you can do the same there instead (return `null`). That makes the code cleaner, and requires less code.

Reply:
Smart :+1: